.github/workflows/ci.yml

```diff
@@ -0,0 +1,116 @@
+name: Build
+
+on:
+  pull_request:
+  push:
+    branches:
+    - master
+    tags:
+    - oss-v*
+
+env:
+  DOTNET_SDK_VERSION: 3.1.201
+jobs:
+  vulnerability-scan:
+    runs-on: ubuntu-latest
+    name: ci/github/scan-vulnerabilities
+    container: mcr.microsoft.com/dotnet/core/sdk:3.1-bionic
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Scan for Vulnerabilities
+      run: |
+        dotnet tool restore
+        dotnet restore
+        dotnet tool run dotnet-retire
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-18.04]
+        test: ["", .Streams, .PersistentSubscriptions, .Operations, .UserManagement, .ProjectionManagement]
+        configuration: [release]
+        docker-tag: ['6.0.0-preview3.21-bionic']
```

Comment:
run against latest and make a separate release build that is repeatable

Reply:
EventStore/EventStore#2407 must get merged first